tests/test_reported_bugs.py

```diff
@@ -30,12 +31,14 @@ def test_issue_455():
         exception_raised = True
         assert ("Tried to call DecodeFrame on a Decoder that has been initialized without a demuxer. "
                 "Please use DecodeFrameFromPacket instead or intialized the decoder with demuxer when decoding "
-                "from a file" == str(ex))
+                "from a file" != str(ex))
```

Comment:
@rnaskulwar this does not make sense. The error message needs to be updated not tested for unequal. Just did that